tests/everflow/test_everflow_testbed.py

```diff
@@ -370,59 +370,178 @@ def test_everflow_remove_used_ecmp_next_hop(self, duthosts, rand_one_dut_hostnam
+            test_name="everflow_policer_test.EverflowPolicerTest",
+            src_port=rx_port_ptf_id,
+            dst_mirror_ports=tx_port_ptf_id,
+            dst_ports=tx_port_ptf_id,
+            meter_type="packets",
+            cir="100",
+            cbs="100",
+            tolerance="10"
+        )
+
+        everflow_utils.remove_route(duthost, policer_mirror_session["session_prefixes"][0], policer_peer_ip)
+
+    def test_two_sessions_same_port(
+        self,
+        duthosts,
+        rand_one_dut_hostname,
+        setup_info,
+        setup_mirror_session,
+        policer_mirror_session,
+        mirror_dscp_table,
+        dest_port_type,
+        ptfadapter,
+        partial_ptf_runner,
+        tbinfo
+    ):
+        """
+        Verify that we can setup two mirror sessions and send traffic to the same monitor port.
+
+        We should be able to send traffic that hits rules in each table, and the traffic should
+        be mirrored to the correct mirror session.
+        """
+        duthost = duthosts[rand_one_dut_hostname]
+
+        # Add a route to the standard mirror session destination IP
+        std_tx_port = setup_info[dest_port_type]["dest_port"][0]
+        std_peer_ip, _ = everflow_utils.get_neighbor_info(duthost, std_tx_port, tbinfo)
+        everflow_utils.add_route(duthost, setup_mirror_session["session_prefixes"][0], std_peer_ip)
+
+        # Add a route to the policer-enforced mirror session destination IP
+        policer_tx_port = setup_info[dest_port_type]["dest_port"][0]
+        policer_peer_ip, _ = everflow_utils.get_neighbor_info(duthost, policer_tx_port, tbinfo)
+        everflow_utils.add_route(duthost, policer_mirror_session["session_prefixes"][0], policer_peer_ip)
+
+        # Verify that mirrored traffic is sent to the standard mirror session
+        rx_port_ptf_id = setup_info[dest_port_type]["src_port_ptf_id"]
+        tx_port_ptf_id = setup_info[dest_port_type]["dest_port_ptf_id"][0]
+        self._run_everflow_test_scenarios(
+            ptfadapter,
+            setup_info,
+            setup_mirror_session,
+            duthost,
+            rx_port_ptf_id,
+            [tx_port_ptf_id]
+        )
+
+        # Verify that the DSCP traffic is sent to the policer-enforced mirror session
+        tx_port_ptf_id = setup_info[dest_port_type]["dest_port_ptf_id"][0]
+
+        partial_ptf_runner(
+            setup_info,
+            policer_mirror_session,
+            self.acl_stage(),
+            self.mirror_type(),
+            expect_receive=True,
+            test_name="everflow_policer_test.EverflowPolicerTest",
+            src_port=rx_port_ptf_id,
+            dst_mirror_ports=tx_port_ptf_id,
+            dst_ports=tx_port_ptf_id,
+            meter_type="packets",
+            cir="100",
+            cbs="100",
+            tolerance="10"
+        )
+
+        everflow_utils.remove_route(duthost, policer_mirror_session["session_prefixes"][0], policer_peer_ip)
```

Comment:
did you need to remove the route to std_peer_ip too?

Reply:
I think you're right, good catch!